components/core/tests/test-locust.sh

```diff
@@ -0,0 +1,23 @@
+#!/bin/bash
+echo Installing locustio
+python3 -m pip install locustio
+
+start_locust(){
+    echo Starting Locust... Locust will not be ready to use until the Docker container starts.
+    locust
+    if [ $? -ne 0 ]; then
+    echo "Error launching Locust. There may be something wrong with your Python install."
+    fi
+}
+
+start_bassa(){
+    echo Building and running container
+    echo Building from: $PWD...
+    docker-compose -f ../../../docker-compose.yml up --build
+    if [ $? -ne 0 ]; then
+        echo "Error launching docker-compose. You may not have docker/docker-compose installed. Please refer to: https://docs.docker.com/install"
+    fi
+}
+
+start_locust &
+start_docker
```

Comment:
You missed out this.

Reply:
This will break the script change it to start_bassa

Reply:
Sorry about the delayed response. It was 4:00am locally, so I just committed the changes. The fix should be in the next commit.